google/cloud/security/common/data_access/dao.py

```diff
@@ -59,8 +59,6 @@
     'cloudsql_ipaddresses': create_tables.CREATE_CLOUDSQL_IPADDRESSES_TABLE,
     'cloudsql_ipconfiguration_authorizednetworks':\
         create_tables.CREATE_CLOUDSQL_IPCONFIGURATION_AUTHORIZEDNETWORKS,
-    'cloudsql_acl_violations':\
```

Comment:
Unrelated to this line, L35 to be exact, could you remove the pylint disables above and improve the doc here as well?